rv2036/rvspoc
# update to 2024

!! Please use **Rebase and merge**!

Local test flow:

1. install node >= 18
2. `npm ci`
3. `npx docusaurus start`
     * only the default locale is available when using docusaurus start, `npx docusaurus start --locale en` to test locale `en`
     * The "Past Championships" dropdown links are not available when using this method
     * **to test full features:**
          ```bash
          npm run build
          cp -a ./archives/2023/Website ./public/2023
          cd ./public
          python -m http.server
          # than check in the browser
          ```
